e2e/tests/integration/api-role-access.ts

```diff
@@ -304,25 +304,25 @@ describe('Restrict access to endpoint by role', () => {
     },
     {
       roles: ["test"],
-      endpoint: "/arrival/234/T1/100",
+      endpoint: "/arrival/234/T1/100/tst",
```

Comment:
is this `/tst` meant to be here? i can't see that this is in any of the /arrival rourtes

Reply:
it's the origin, I'll call it origin.